apps/talon/talon_debug/talon_debug.talon

```diff
@@ -0,0 +1,16 @@
+app: talon_debug
+-
+tag {user.talon_tags}: user.paste("{talon_tags}")
+
+#commands for dictating key combos
+key <user.keys> over: "{keys}"
+key <user.modifiers> over: "{modifiers}"
+
+action {user.talon_actions}: insert(talon_actions)
+# requires user.talon_populate_lists tag. do not use with dragon
+list {user.talon_lists}: "{talon_lists}"
+
+# requires user.talon_populate_lists tag. do not use with dragon
+capture {user.talon_captures}: "{talon_captures}"
+set {user.talon_settings}: user.paste("{talon_settings}")
+application {user.talon_apps}: user.paste("{talon_apps}")
```

Comment:
We will probably recommend to remove user.paste and insert and just use the literal for all insertions. It's a bit confusing that we are inserting text in three different ways now.